### DISCUSSION TITLE
Student developer pack

### DISCUSSION THREAD

**Discussion by ghost** [community/community#60630] [Packages]
My student developer pack in the portal got approved but my benefits are not available yet. My project deadline is approaching

**Comment by anhtran917** [comment]
I think you need to reach out to the support team of the platform or organization that approved your student developer pack. 
Explain your situation and provide any relevant details, such as your approval confirmation or any reference numbers.
They will assist you further and provide an update on the status of your benefits.

**Reply by ghost** [reply]
How can I reach them? Can you provide me the contact number or mail?

**Reply by anhtran917** [reply]
Maybe they can help you.
Submit your issue to here.
[Ask to Github Support](https://support.github.com/contact/account?tags=dotcom-footer%2Chubberfy_packages)
